recipes/calculix/Makefile_MT

```diff
@@ -0,0 +1,30 @@
+AR ?= ar
+FC ?= gfortran
```

Comment:
@looooo, FC is set here.

Reply:
@isuruf No without setting it explicitly in the bld.bat it didn't work and f77 was used.